config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -2526,6 +2527,79 @@ public HttpSecurity httpBasic(Customizer<HttpBasicConfigurer<HttpSecurity>> http
 		return HttpSecurity.this;
 	}
 
+	/**
+	 * Configures Well-Known URL for Changing Passwords.
+	 *
+	 * <h2>Example Configuration</h2>
+	 * The example below demonstrates how to configure Well-Known URL for Changing Passwords for an
+	 * application. The default change password URL is "/change-password", but can be
+	 * customized using {@link WellKnownChangePasswordConfigurer#changePasswordUrl(String)}.
+	 *
+	 * <pre>
+	 * &#064;Configuration
+	 * &#064;EnableWebSecurity
+	 * public class WellKnownChangePasswordSecurityConfig extends WebSecurityConfigurerAdapter {
+	 *
+	 * 	&#064;Override
+	 * 	protected void configure(HttpSecurity http) throws Exception {
+	 * 		http
+	 * 			.authorizeRequests(authorizeRequests ->
+	 * 				authorizeRequests
+	 * 					.antMatchers(&quot;/**&quot;).hasRole(&quot;USER&quot;)
+	 * 			)
+	 * 			.wellKnownChangePassword();
+	 *    }
+	 * }
+	 * </pre>
+	 *
+	 * @return the {@link WellKnownChangePasswordConfigurer} for further customizations
+	 * @throws Exception
+	 * @see <a href="https://wicg.github.io/change-password-url/index.html">Well-Known URL for Changing Passwords</a>
+	 */
+	public WellKnownChangePasswordConfigurer<HttpSecurity> wellKnownChangePassword() throws Exception {
```

Comment:
I think this might be too narrow at this level of the DSL. Given https://github.com/spring-projects/spring-security/issues/7395, I wonder if a better setup would be:

```java
http
    .passwordManagement(pass -> pass
        .changePasswordPage("/my-page")
    );
```

This seems like a good fit for other password management components down the road:

```java
http
    .passwordManagement(pass -> pass
        .changePasswordPage("/my-page")
        .changePasswordProcessingUrl("/my-processing-url")
        .passwordChecker(...)
        .passwordEncoder(...)
        // ...
    );
```

I like `changePasswordPage` since that aligns with `loginPage`

Reply:
Thank you for the review, @jzheaux 
Yes, the `passwordManager` seems to be a better option, so I'll update my PR.